packages/gatsby-source-drupal/src/normalize.ts

```diff
@@ -154,7 +158,12 @@ export const nodeFromData = async (
     typeof attributeId !== `undefined` ? { _attributes_id: attributeId } : {}
 
   const langcode = attributes.langcode || `und`
-  const type = datum.type.replace(/-|__|:|\.|\s/g, `_`)
+  const { typePrefix = `` } = pluginOptions
+  const cleanedType = datum.type.replace(/-|__|:|\.|\s/g, `_`).replace(/^_/, ``)
```

Comment:
This strips leading underscores from type names, which could technically be a breaking change. I'm not sure if it is in practice?

Reply:
Leading underscores are allowed so this is a breaking change: https://spec.graphql.org/June2018/#sec-Names

Why do you need to strip them?